spec/helpers/helpers.rb

```diff
@@ -0,0 +1,11 @@
+
+  def sign_up_as(user) 
```

Comment:
Assignment Branch Condition size for sign_up_as is too high. [17/15]<br>Trailing whitespace detected.